test/test_core.py

```diff
@@ -6878,7 +6877,7 @@ def test_freetype(self):
   })
   def test_sqlite(self, use_pthreads):
     if use_pthreads:
-      self.set_setting('USE_PTHREADS')
+      self.emcc_cflags.append('-pthread')
```

Comment:
What's the difference between `emcc_cflags` and `emcc_args`?

Reply:
Oh good catch.. emcc_cflags doesn't exist :)